### ISSUE TITLE
[REQUEST] Add Block Variant support in Harvester

### ISSUE THREAD

**Issue by hellbringer616** [OnapleRPG/Harvester#13]
Allow for variant blocks to drop themselves as items if desired; see example below.
```
harvest_items = [
  {
    type: "minecraft:stone",
    state {
        variant { value = "diorite" }
    },
    pool_ref: 1
  }
]
``

**Comment by hellbringer616** [comment]
Could potentially so solved by #12 if you can drop the default item AND the defined items from `harvest_items`

**Comment by zessirb** [comment]
It looks like two different behaviors to me; but both seem interesting. 
We will most likely focus on #12 first then handle the variant.

**Comment by hellbringer616** [comment]
Sounds good! basically i am trying to make it so Granite and Diorite drop itemizer items AND themselves (at least sometimes) and all i get is stone... I imagine this will be fixed by #12 but if i am wrong; so be it :)

**Comment by zessirb** [comment]
Indeed, for your need only one of those will suffice. We might want to allow variant drop anyway though (like dropping diorite for dirt or something similar), so I believe this feature would be nice too :)

**Comment by zessirb** [comment]
You should be able to do that now. You can't directly specify the item variant in the harvest_items item_name as of now, but you can bind it with Itemizer, and the default drop feature has been implemented too.  
(Available from [1.2.0](https://github.com/OnapleRPG/Harvester/releases/download/1.2.0/Harvester.jar) and onward)
